tests/integration-tests/tests/dashboard/test_dashboard/test_dashboard/pcluster.config.yaml

```diff
@@ -0,0 +1,26 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+Scheduling:
+{% if {{ scheduler }} == "awsbatch" %}Awsbatch:{% else %}Slurm:{% endif %}  # FIXME: need to make it camelcase
+    Queues:
+      - Name: queue1
+        Networking:
+          SubnetIds:
+            - {{ public_subnet_id }}
+        ComputeResources:
+          - Name: compute_resource1
```

Comment:
`compute-resource1. Underscore is not accepted

Reply:
Thanks! Fixed. Will fix and merge this after adapting the schema